Author: Adam Gandelman <adamg@canonical.com>
Date: Thu Oct 11 00:42:53 PDT 2012
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/nova/+bug/1059907
Subject: Avoid installation of python-setuptools-git.

Remove setuptools_git from pip-requires so that it is not automatically
added to the dependency list by dh_python2, and avoid having it and its
dependencies (git!) installed with python-nova.

Index: nova-2012.2/tools/pip-requires
===================================================================
--- nova-2012.2.orig/tools/pip-requires	2012-10-11 00:36:41.000000000 -0700
+++ nova-2012.2/tools/pip-requires	2012-10-11 00:38:28.649147690 -0700
@@ -18,6 +18,5 @@
 Babel>=0.9.6
 iso8601>=0.1.4
 httplib2
-setuptools_git>=0.4
 python-quantumclient>=2.0
 python-glanceclient>=0.5.0,<2
